tests/structures/test_assignment.py

```diff
@@ -69,7 +71,37 @@ def test_list_assignment(self):
             print(x)
             print(y)
             print(z)
-            """)
+        """)
+
+    @expectedFailure
+    def test_bad_list_assignment(self):
+        self.assertCodeExecution("""
+            try:
+                [x, y, z, a] = range(3)
+                print(x)
+                print(y)
+                print(z)
+                print(a)
+            except ValueError as e:
+                print("Got a ValueError:", e)
+        """)
+
+    def test_bad_list_assignment_raises_StopIteration(self):
+        """
+        This test is a copy of test_bad_list_assignment.
+        Raising ValueError as a result of a bad unpacking is not currently implemented;
+        this test should be deleted when it is.
+        """
+        self.assertCodeExecution("""
+            try:
+                [x, y, z, a] = range(3)
+                print(x)
+                print(y)
+                print(z)
+                print(a)
+            except Exception as e:
```

Comment:
In retrospect, it's probably better to specifically catch `StopIteration` - that way the test will fail if/when we get around to fix the underlying problem, *this* test will fail, but hopefully the documentation will point at the way to make the test pass - to delete it :-)

Reply:
Yes it would be -- but then the test will fail because there will be no `StopIteration` to catch on the Python side. Or are you suggesting to use `except (StopIteration, ValueError) as e:`? I think that when this problem is fixed, the correct test below will result in an unexpected pass, and the comment there should be enough to point the human to delete this test?

Reply:
Sorry - yes, I meant to catch `(StopIteration, ValueError) as e`

Reply:
Done! :-)